.github/workflows/main.yaml

```diff
@@ -129,6 +146,39 @@ jobs:
           body_path: ./release-artifacts/changelog.md
           draft: ${{ startsWith(steps.get_tag.outputs.git_tag, 'nightly') != true }}
           prerelease: ${{ startsWith(steps.get_tag.outputs.git_tag, 'nightly') }}
+  push-image:
+    name: Push Image
+    needs:
+      [create-release]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: |
+            kubent:${{ github.sha }},
+            kubent:${{ needs.create-release.outputs.tag_name }}
```

Comment:
I would worry this might not work - it's like a mix of both the supported formats, see [^1]. I think it has to be either without comas or one-liner.


[^1]: https://github.com/docker/build-push-action#inputs

Reply:
Also based on the examples in readme [^1], the GH user name (or org name in our case) is missing in the tag names. Perhaps the earlier introduced `IMAGE_NAME` was intended to be used here?

[^1]: https://github.com/docker/build-push-action